### ISSUE TITLE
Tokomak can't find rustup during install, but rustup is there

### ISSUE THREAD

**Issue by nathanaeljones** [vertexclique/tokamak#70]
```Failed to spawn command $HOME/.cargo/bin/rustup. Make sure $HOME/.cargo/bin/rustup is installed and on your PATH

>  $HOME/.cargo/bin/rustup
rustup 1.0.0 (17b6d21 2016-12-15)

```

**Comment by vertexclique** [comment]
If rustup is not in your PATH it can't be detected.

**Comment by nathanaeljones** [comment]
It's in $PATH.

**Comment by nathanaeljones** [comment]
Sorry, just realized markdown was hiding the error I reported. Inserted newline.

**Comment by vertexclique** [comment]
can you expand the path and try again? like:

```/home/nathanaeljones/.cargo/bin/rustup```

**Comment by nathanaeljones** [comment]
This is only during package installation. It works afterwards, and in the terminal.

**Comment by vertexclique** [comment]
Cool, will take a look at it. By default if it couldn't find that, it defaults to `$HOME/.cargo/bin/rustup`. I don't know why was the reason but i will try to reproduce. Thanks for feedback.

**Comment by nathanaeljones** [comment]
Yeah, `$HOME/.cargo/bin/rustup` is where rustup is located. I'm using nightly-2017-03-04, in case that matters.

**Comment by nullterminated** [comment]
I had this same problem. In my .bashrc file I had put

export PATH=$PATH:~/.cargo/bin

The problem went away when I changed that to

export PATH=$PATH:$HOME/.cargo/bin

**Comment by vertexclique** [comment]
Ok, this looks like tilde expansion problem. By default, I bypassed this by using `$HOME` variable.
